src/Microsoft.Identity.Client/TelemetryCore/TelemetryTokenCacheAccessor.cs

```diff
@@ -45,6 +45,12 @@ public TelemetryTokenCacheAccessor(ITokenCacheAccessor tokenCacheAccessor)
         }
 
 #if iOS
+        public void SetiOSKeychainSecurityGroup(string keychainSecurityGroup)
+        {
+            _tokenCacheAccessor.SetiOSKeychainSecurityGroup(keychainSecurityGroup);
+        }
+
```

Comment:
should this be in the interface as well?

Reply:
It is in the ITokenCacheAccessor...should it be somewhere else as well? I'm not getting any build errors.

---
In reply to: [246884089](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/771#discussion_r246884089) [](ancestors = 246884089)